libshpool/src/daemon/server.rs

```diff
@@ -381,6 +382,7 @@ impl Server {
                 if let Err(err) = self.hooks.on_shell_disconnect(&header.name) {
                     warn!("shell_disconnect hook: {:?}", err);
                 }
+                let _s = span!(Level::INFO, "2_lock(shells)").entered();
```

Comment:
neat, read about [span](https://docs.rs/tracing/latest/tracing/span/index.html)

I love how dropping the variable triggers `.exit()` and can be used to measure narrow contexts. this is very clean